src/server/views/help/cookies.html

```diff
@@ -2,37 +2,40 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
+{% set pageTitle = "Cookies" %}
+
 {% block content %}
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">Cookies on {{ serviceName }}</h1>
-        <p class="govuk-body">Cookies are files saved on your phone, tablet or computer when you visit a website.</p>
-        <p class="govuk-body">We use cookies to remember information you've entered when applying to prove your eligibility.</p>
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Cookies</h1>
+    <p class="govuk-body">Cookies are small files saved on your phone, tablet or computer when you visit a website.
+    </p>
+    <p class="govuk-body">We use essential cookies to make this form work.</p>
 
-        <h2 class="govuk-heading-l">Strictly necessary cookies</h2>
-        <h3 class="govuk-heading-m">Your progress when using this service</h3>
-        <p class="govuk-body">When you use this service, we’ll set a cookie to remember your progress through the forms. These cookies do not store your personal data and are deleted once you’ve completed the transaction.</p>
-        <table class="govuk-table">
-            <thead>
-                <tr class="govuk-table__row">
-                    <th class="govuk-table__header" scope="col">Name</th>
-                    <th class="govuk-table__header" scope="col">Purpose</th>
-                    <th class="govuk-table__header" scope="col">Expires</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">session</td>
-                    <td class="govuk-table__cell">Set to remember information you’ve entered into a form</td>
-                    <td class="govuk-table__cell">When you close your browser</td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">crumb</td>
-                    <td class="govuk-table__cell">Helps us to prevent cross site scripting attacks</td>
-                    <td class="govuk-table__cell">When you close your browser</td>
-                </tr>
-            </tbody>
-        </table>
-    </div>
+    <h2 class="govuk-heading-m">Essential cookies</h2>
+    <p class="govuk-body">Essential cookies keep your information secure. We do not need to ask your permission to
+      use them.</p>
+    <table class="govuk-table">
+      <thead>
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">Name</th>
+          <th class="govuk-table__header" scope="col">Purpose</th>
+          <th class="govuk-table__header" scope="col">Expires</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell">session</td>
```

Comment:
This was missed from the previous HTML but can we fix an accessibility issue?

Without the [`govukTable()` macro](https://design-system.service.gov.uk/components/table/) it's easy to miss cookie names “session” and “crumb” as headings

Unlike the column headings they'll need `scope="row"`

```suggestion
          <th scope="row" class="govuk-table__header">session</th>
```